File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.py
class Solution:
    def finalValueAfterOperations(self, op: List[str]) -> int:
        ans = 0;
        for st in op:
            if(st == "++X" or st == "X++"):
                ans+=1;
            if(st == "--X" or st == "X--"):
                ans-=1;
        return ans;
        

File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.cpp
class Solution {
public:
    int finalValueAfterOperations(vector<string>& op) {
        int ans = 0;
        for(string str : op){
            if(str == "++X" || str == "X++") ans++;
            if(str == "--X" || str == "X--") ans--;
        }
        return ans;
    }
};